tutorcelery/patches/k8s-deployments

```diff
@@ -31,6 +31,9 @@ spec:
             - "--hostname=edx.{{service}}.core.{{variant}}.%%h"
             - "--max-tasks-per-child=100"
             - "--queues=edx.{{service}}.core.{{variant}}"
+            {% for setting, value in config.get("extra_params", {}).items() %}
+            - "--{{setting}}={{value}}"
+            {% endfor %}
```

Comment:
Maybe we can have the logic for the command in the python code and use a template filter to render the final command.

Reply:
I prefer rendering this code on the template, it's just a setting and a value